pkg/web/apps/apps_psql_repository.go

```diff
@@ -19,16 +22,59 @@ func NewPostgreSQLRepository(db *sql.DB) Repository {
 
 // GetApps retrieves all apps from the database
 func (a *appsPostgreSQLRepository) GetApps() (*[]models.App, error) {
-	app1 := models.App{
-		ID:                    "a0874c82-2b7f-11e9-b210-d663bd873d93",
-		AppID:                 "com.aerogear.app1",
-		AppName:               "app1",
-		NumOfDeployedVersions: 1,
-		NumOfClients:          1,
-		NumOfAppLaunches:      1,
+	rows, err := a.db.Query(`SELECT id,app_id,app_name FROM app WHERE deleted_at IS NULL;`)
+
+	if err != nil {
+		log.Error(err)
+		return nil, models.ErrInternalServerError
 	}
 
-	apps := []models.App{app1}
+	defer func() {
+		err := rows.Close()
+		if err != nil {
+			log.Error(err)
```

Comment:
This come after the SQL data has been scanned into the list of Apps. The error is because the connection to the database could not be closed, but we still got the apps and they will be returned at the end. Maybe this should be changed to a warning log?

Reply:
@davidffrench This come after the SQL data has been scanned into the list of Apps. The error is because the connection to the database could not be closed, but we still got the apps and they will be returned at the end. Maybe this should be changed to a warning log?

Reply:
Thanks for the clarification. I think it should stay as an Error in this case.